core/src/main/java/com/spotify/ffwd/debug/NettyDebugServer.java

```diff
@@ -132,38 +119,32 @@ private void sendInspectPacket(Object event) throws Exception {
 
         s.childHandler(new ChannelInitializer<Channel>() {
             @Override
-            protected void initChannel(final Channel ch) throws Exception {
+            protected void initChannel(final Channel ch) {
                 connected.add(ch);
                 log.info("Connected {}", ch);
 
-                ch.closeFuture().addListener(new ChannelFutureListener() {
-                    @Override
-                    public void operationComplete(ChannelFuture future) throws Exception {
-                        connected.remove(ch);
-                        log.info("Disconnected {}", ch);
-                    }
+                ch.closeFuture().addListener((ChannelFutureListener) future1 -> {
```

Comment:
call this `f` or `future`? you use `f` below.

Reply:
Renamed it to `f` (this was intellij's auto-refactoring btw).